src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import duke.tasks.Task;
+
+
+/**
+ * Class containing the task list.
+ */
+public class TaskList {
+
+    private ArrayList<Task> list;
```

Comment:
Consider naming `list` as something else that's more descriptive? like `tasks` or something along those lines.